tests/browser/async_2.cpp

```diff
@@ -7,7 +7,7 @@
 #include <emscripten.h>
 #include <assert.h>
 
-int calc(int x) {
+__attribute__((noinline)) int calc(int x) {
```

Comment:
Is this line alone not enough to fix the issue?

Reply:
No, since it doesn't apply to binaryen sadly...

Reply:
(but this might prevent future regressions if LLVM wants to try to inline it...)